script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -205,15 +205,15 @@ But the teachers rely on you because you do that. I hope you don't let them get
 <Page63>
 "It's okay, I do it because I like it.# Besides, the teachers are thinking about the school. I don't like the way you're talking about them, Inui-kun."
 <Page64>
-Ciel-senpai rebukes Arihiko's lazy opinions.
+Ciel-senpai completely rebukes Arihiko's lazy opinions.//だらけきった有彦の意見を、正面からばっさりたしなめるシエル先輩。
 From the looks of it, he seems to be close to her in his own way.
 <Page65>
 "Senpai, do you do volunteer work every day by any chance?"
 "What, you don't know? Ciel-senpai here is such a handy person that she's called the true student council president."
 <Page66>
-"I know. She's the perfect third year who can solve anything, unlike the student council which is just a title at best."
+"I know. She's the perfect third year who can solve anything, unlike the so-called student council."//知ってる。肩書きだけの生徒会と違って、なんでも解決してくれる完璧な三年生だろ」If anyone has better ideas on how to phrase this
 <Page67>
-"Yeah. There's even a fan club made among the first years, and whenever something troublesome happens, even the teachers would say, 'Just talk to Ciel and you'll be fine'. Nowadays, Senpai could do just about anything and the teachers wouldn't complain."
+"Yeah. There's even a fan club made among the first years, and whenever something troublesome happens, even asking the teachers would get you nothing more than a 'Just talk to Ciel and you'll be fine'. Nowadays, Senpai could do just about anything and the teachers wouldn't complain."//「おう。一年じゃファンクラブまであるし、教師連中も厄介ごとが起こると『シエルに相談すれば安心だ』なんて<暖簾|のれん>に腕押し状態だからな。今じゃ先輩が何をしたって文句を言う教師もいないぐらいだ」 
```

Comment:
Unrelated to your change, but don't love the 2x `even` here, maybe drop the first one.